packs/data/pfs-season-3-bestiary.db/enraged-shrouded-bloodseeker.json

```diff
@@ -63,7 +63,7 @@
             "privateNotes": "",
             "publicNotes": "Scourges of swamps and damp, abandoned places, bloodseekers are ravenous blood drinkers. Farmers curse the creatures for sucking their livestock dry. It is from such beleaguered people that the bloodseeker's regional name \"stirge\", possibly a corruption of the word \"scourge\", comes. Folk wisdom holds that the appearance of bloodseekers in a region signals a healthy herd of livestock, but more often it means bogs or old buildings that haven't been properly tended to. Certainly, no amount of folksy parable can assuage a farmer driven to destitution by a bloodseeker infestation. But despite their role as parasites, bloodseekers aren't hated by all villages. In some cases, the inhabitants of remote backwoods thorps even keep the things as pets or use them as doubtful medicinal \"tools\" to drain away unwanted humors or test for evil spirits possessing the blood. Worshippers of gods of pestilence and parasites often view bloodseekers as sacred to their faith and allow the creatures to feed freely from their bodies. In such societies, those who accidentally give too much are considered to have been \"blessed\" by the village's hungry god.</p>\n<p>Bloodseekers seem to be constantly hungry, but they are not inherently malevolent. They can be scared away fairly easily and prefer to swiftly retreat rather than risk death. Some adventurers report that these creatures can be scared away by waving torches at the flying pests. However, bloodseekers are much bolder when encountered in larger numbers, as bringing down one victim lets an entire colony feed. Bloodseeker colonies are called clots, for obvious and disgusting reasons. If a lone bloodseeker finds a likely victim while its clot is nearby, it emits a high-pitched, keening noise to summon reinforcements.</p>\n<p>Most humanoids avoid bloodseekers, but boggards sometimes cultivate bloodseeker nests around the perimeter of their territory. These colonies serve as a deterrent to intruders, and the boggards sometimes check for bloodseeker prey, collecting the hides or bodies of animals killed by the pests. Meals prepared from slain bloodseekers that have gorged on the blood of specific creatures are a staple among certain boggard communities. The boggards not only eat the actual bloodseekers, but they also make a gelled slurry from the drained blood.</p>\n<p>A typical bloodseeker is about a foot long, with mottled, reddish-brown skin and a yellow underbelly. Its four wings resemble bat wings. When gorged with blood, the creature becomes bloated and pink, and it tends to wobble unsteadily in the air as it flies off to digest its meal.",
             "source": {
-                "value": "Pathfinder Society Intro: Year of Shattered Sanctuaries"
+                "value": "Pathfinder Society Intro #3: Year of Shattered Sanctuaries"
```

Comment:
Why #3? That isn't part of the title: https://paizo.com/products/btq027s9?Pathfinder-Society-Intro-Year-of-Shattered-Sanctuaries

Reply:
```suggestion
                "value": "Pathfinder Society Intro: Year of Shattered Sanctuaries"
```